modules/applicationelb-dashboard/main.tf

```diff
@@ -22,7 +22,7 @@ resource "lightstep_metric_dashboard" "aws_app_elb_dashboard" {
       display    = "bar"
       hidden     = false
 
-      metric              = "aws.application_elb.request_count_count"
+      metric              = "aws.elb.request_count_count"
```

Comment:
It should be `aws.applicationelb`, AWS CloudWatch Metrics for your Application Load Balancer. `aws.elb.metricname` is AWS CloudWatch Metrics for your Classic Load Balancer.

Reply:
nice. ty